src/openrct2/management/NewsItem.h

```diff
@@ -36,7 +36,8 @@ namespace News
         Peeps,
         Award,
         Graph,
-        Count
+        Count,
+        Campaign
```

Comment:
Count has to be the last entry of the enum, so ensure that its value encapsulates all others.

```suggestion
        Campaign,
        Count
```

This will result in some static_asserts being hit, which are there to make sure the rest of the code behaves as expected.

Reply:
Thanks, I didn't realize that. Just so used to adding a new enum value at the end :-).